### PR TITLE
contract implementation

### PR DESCRIPTION
**Pull request by wgcisotto**
contract & scripts

### CODE REVIEW

**Review by kasmaldev — APPROVED** [review]
Approved William. Good work . Thanks

**Comment by kasmaldev** [comment]
Approved William. Good work . Thanks

File: Contract/contracts/ERC721.sol [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
```
Comment (web3allday) [inline Contract/contracts/ERC721.sol:6]: import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Burnable.sol"; missing

Comment (web3allday) [reply Contract/contracts/ERC721.sol:6]:
```suggestion
import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Burnable.sol"; 
import "@openzeppelin/contracts/access/Ownable.sol";
```

Comment (wgcisotto) [reply Contract/contracts/ERC721.sol:6]: Hi Rhen, thanks for pointing this out, the classes ERC721 and ERC721URIStorage have the burn function, I don't need to import the Burnable, however, I noticed that I forgot to add a public method to burn by tokenId, I'm creating a new PR for that! thanks!

Comment (web3allday) [reply Contract/contracts/ERC721.sol:6]: Thanks for letting me know William!

**Pull request merged**